README.md

```diff
@@ -47,6 +48,16 @@ Minimum system requirements: C++11 enabled compiler, [cmake](http://cmake.org),
 
 DGtal can be compiled on Microsoft Windows system using Visual Studio 2014 (or newer): Generate the Visual Studio project using windows [cmake](http://cmake.org) tool and compile the DGtal solution (you may also need to set the cmake variable ```BUILD_SHARED_LIBS``` to false).
 
+
+License
+=======
+
+DGtal core libarary is under the GNU LGPL v3 license (see [LICENSE.txt](LICENSE.txt)). DGtal documentation and website materials are under the [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 ](http://creativecommons.org/licenses/by-nc-sa/4.0/).
+
+If some dependencies are activated, resulting binaries may follow the dependencies licenses. More precisely:
+* If the library is built with the 3D internal viewer using [libQGLViewer](http://libqglviewer.com) (```WITH_QGLVIEWER``` flag set to ```true```), then resulting binaries libQGLViewer license (GNU GPL or commercial, see [libQGLViewer licensing](http://libqglviewer.com/dualLicensing.html)).
```

Comment:
it looks something is missing at the end:
=> "then the resulting binaries are under the libQGLViewer license (GNU...)"?